frontend/src/components/Auth/index.js

```diff
@@ -0,0 +1,4 @@
+import { AuthContext, AuthProvider } from './AuthContext.js';
+import PrivateRoute from './PrivateRoute.js';
+
+export { AuthContext, AuthProvider, PrivateRoute };
```

Comment:
Another thing I am curious about in terms of your design decisions. Did you choose to make AuthContext.js in a separate file rather than in index.js so that you could import them all from ../Auth in a clean way or was it something else?

Reply:
I did it this way because there were several important components I wanted to use out of the Auth directory, and thought it would be cleaner to have each of them in their own file with the index.js file used simply to export them. There are probably other way of structuring it too.